src/structures/IntegrationApplication.js

```diff
@@ -15,6 +15,48 @@ class IntegrationApplication extends Application {
      * @type {?User}
      */
     this.bot = data.bot ? this.client.users.add(data.bot) : this.bot ?? null;
+
+    /**
+     * The url of the app's terms of service
+     * @type {?string}
+     */
+    this.termsOfServiceURL = data.terms_of_service_url ?? this.terms_of_service_url ?? null;
+
+    /**
+     * The url of the app's privacy policy
+     * @type {?string}
+     */
+    this.privacyPolicyURL = data.privacy_policy_url ?? this.privacyPolicyURL ?? null;
+
+    /**
+     * The Array of RPC origin urls
+     * @type {string[]}
+     */
+    this.rpcOrigins = data.rpc_origins ?? this.rpcOrigins ?? [];
+
+    /**
+     * The application summary
+     * @type {?string}
+     */
+    this.summary = data.summary ?? this.summary ?? null;
+
+    /**
+     * Whether the application has been associated with a webhook
+     * @type {?boolean}
+     */
+    this.hook = data.hook ?? this.hook ?? null;
```

Comment:
this is incorrect
> This is mainly for the client. And related to games.
> 
> Hook controls whether or not the overlay and other things are allowed to default hook a game. Let's us turn it off quickly if it's causing crashes.

https://github.com/discord/discord-api-docs/issues/2149#issuecomment-707967604